src/Haskell/Ide/Engine/Support/Hoogle.hs

```diff
@@ -22,27 +22,6 @@ import           Text.HTML.TagSoup.Tree
 
 -- ---------------------------------------------------------------------
 
-hoogleDescriptor :: PluginId -> PluginDescriptor
```

Comment:
Hoogle Plugin does not exist anymore? Out of curiosity, what does that mean? Was it previously possible to invoke the PluginCommands "info" and "lookup" without a hover request? Or is this change unobservable for the lsp client?

Reply:
Through LSP no, it was for when the JSON transport could run plugin commands directly. The functionality is still there in the `lookup`/`info` cmds, just not registered as a command.